src/components/PopupDelete.js

```diff
@@ -0,0 +1,20 @@
+import Popup from "./Popup.js";
+
+export default class PopupDelete extends Popup {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.

Reply:
Thank you!